config/initializers/job_runner.rb

```diff
@@ -0,0 +1,3 @@
+unless Rails.env.test? || !ActiveRecord::Base.connection.table_exists?('spina_registers') || File.basename($0) == 'rake'
+  PopulateRegisterDataInDbJob.perform_now
```

Comment:
Could we move this into `production.rb` instead of having the conditional clause?

Reply:
Actually we want to run this at the moment on `development` also. It was just breaking tests and db migration if run in those circumstances I will see if I can refactor this, it will be replaced by a proper scheduled task in due course.